api/src/test/java/bio/terra/tanagra/aousynthetic/GeneratedSqlUtils.java

```diff
@@ -0,0 +1,84 @@
+package bio.terra.tanagra.aousynthetic;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.PrintWriter;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Path;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Utility class for:
+ *
+ * <p>- comparing the SQL generated by the API code against an expected string saved in a golden
+ * file.
+ *
+ * <p>- overwriting the existing golden file with a new expected string.
+ */
+public final class GeneratedSqlUtils {
```

Comment:
nit: not necessarily specific to aousynthetic, could potentially live in a more general location. Fine to also just move later once we have a second use case.

Reply:
I was planning to just move it once we have a second use case, but then I noticed there's a generic-sounding `testing` package. Moved it there instead.